### ISSUE TITLE
[Issue]: How to Repro GAIA Benchmark Results

### ISSUE THREAD

**Issue by CrazySherman** [microsoft/autogen#2592]
### Describe the issue

Hi, I'm wondering if there's a readme or notebook to repro GAIA results, trying to dig in the code but not able to find it, thx!

**Comment by ekzhu** [comment]
Cc @afourney @gagb

**Comment by afourney** [comment]
Yes, I need to write a proper readme. There are some instructions here: https://github.com/microsoft/autogen/issues/1943#issuecomment-1987278247

That should hopefully be sufficient to get you started, but let me know if you run into trouble.
